GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/RO_BDB_AtlasV.cfg

```diff
@@ -14,7 +14,7 @@
 	%thermalMassModifier = 1.0
 	%skinMassPerArea = 4
 
-	!RESOURCE, * {}
```

Comment:
I'm not entirely sure that this is invalid. Regardless it's good to normalize.

Reply:
That was my thought as well. MM is pretty permissive in what it handles, so I'm not certain that it wouldn't handle `, *`, but I do know for sure that `,*` **is** valid.